### ISSUE TITLE
Passing uploadFolder as a parameter to cli does nothing

### ISSUE THREAD

**Issue by yaroslavlvov** [midudev/s3-folder-upload#12]
when I try to upload folder using cli it always uploads it in the root of aws s3. I set parameter like
--uploadFolder=myFolder

**Comment by midudev** [comment]
When using like CLI, you pass the folder as first parameter.

```
s3-folder-upload <folder>
```

So, in order to upload `statics` folder, you would do:
```
s3-folder-upload statics
```

You could pass some options, for example, for credentials :
```
s3-folder-upload statics --accessKeyId=<your access key id> --bucket=<destination bucket> --region=<region> --secretAccessKey=<your secret access key>
```

**Comment by IniZio** [comment]
Its actually because the `uploadFolder` was not passed in allowed fields. I made an adhoc fix in fork https://github.com/IniZio/s3-folder-upload/commit/24a1426d8acad07d4e283fe1142f68c12c56098f.

**Comment by midudev** [comment]
I've published version 2.3.5 with the fix. Thanks @IniZio 👏
